pages/Real-world usage.md

```diff
@@ -61,17 +61,24 @@ For example, we can start five Redix processes under our supervision tree and
 name them `:redix_0` to `:redix_4`:
 
 ```elixir
+# lib/my_app.ex
```

Comment:
Can you remove this? This code can be anywhere, and while `lib/my_app.ex` would probably be the best place, I think it adds more noise to this section. Wdyt?